public/locales/en/common.json

```diff
@@ -56,7 +56,7 @@
   "cancel": "Cancel",
   "selectCurrency": "Select Currency",
   "donationSuccessfulWith": "Your {{currency}} {{totalAmount}} donation was successfully paid with {{paymentTypeUsed}}.",
```

Comment:
If we add the frequency to `donationSuccessful`, shouldn't we also add it to `donationSuccessfulWith`?

Reply:
- [ ] Just trying to put the word for frequency inside the resource does not alway work in every language, e.g. for German you probably want to translate "yearly" with "jährlich" when choosing the frequency on the first screen, but in the following context your have to write "jährliche" - otherwise it is just wrong - it's also not really possible to change the wording to use "jährlich" - we need another solution here:

<img width="394" alt="Bildschirmfoto 2021-11-16 um 18 08 36" src="https://user-images.githubusercontent.com/1532418/142032148-32296798-61f0-4b79-bfc1-4747b666af1e.png">